runner/runner.go

```diff
@@ -399,7 +397,3 @@ func (r *Runner) canStartServer() bool {
 func (r *Runner) canStartWarehouse() bool {
 	return r.appType != app.GATEWAY && r.warehouseMode != config.OffMode
 }
-
-func (r *Runner) canStartBackendConfig() bool {
-	return r.warehouseMode != config.SlaveMode
-}
```

Comment:
So with these changes the warehouse slaves will be able to start pulling backend configurations. What's the rationale behin this decision? We could use some info in the PR description.